### PR TITLE
finish init project

### PR DESCRIPTION
**Pull request by xtrxllx**

### CODE REVIEW

**Comment by xtrxllx** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.0'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (manhnt-0960) [inline Gemfile]: pls remove all comment

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+gem 'rails', '~> 5.2.3'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
+# Use Puma as the app server
+gem 'puma', '~> 3.11'
```
Comment (manhnt-0960) [inline Gemfile]: let's use `"` instead of `'`

File: Gemfile (hunk 1 above)
Comment (xtrxllx) [reply Gemfile]: Ready

File: Gemfile (hunk 2 above)
Comment (xtrxllx) [reply Gemfile]: Ready

**Comment by xtrxllx** [comment]
ready

**Review by chungpv-1008 — COMMENTED** [review]
Please only leave 1 commit when pushed.

**Comment by xtrxllx** [comment]
ready

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+ruby-2.7.0
```
Comment (saobv-2223) [inline .ruby-version]: image

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,46 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.0'
+
```
Comment (saobv-2223) [inline Gemfile]: using "

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb:9]: using "

File: config/credentials.yml.enc [hunk 6]
```diff
@@ -0,0 +1 @@
+kIsfOxyZ/9B8is5qEDTJl5ckWdSvvzD9pPRKSSl9BGZCoNcZpI2Y3AmRlr1PMwe696KJyIYBebxkQXjLSWaVXT8HEw2kDjWgVTiR/fXL1bmuycMCi56Ih0kVWsaIgwqrAJ/fkDiDboGcV95GmEU6ugew0z9BuDemU9w/+BYaaWA3X2zbDBcsGxY9NV2lsA/khrFp1jlCXVp1bdFYOvzg3z4DnO0VaCzx0W4+pWZWYB3B6v5rw+cbabBKfcMGJMGz0mlBn8BhWBC86ioNeBHDSLBxzRBkttkbooeSBhp6o7qZAYt43d/z7rhS8mJzHJHzxZdSGm3jiDMcWc1fplQlvBBfQwuR4gAhkUFd9iUMR2qTOr2MLr4RAyCdzBtOrcIm6Uh8tq3OB8xO0bRW9qub6uwTff+8vfIzx6JY--gmKaed8ZYBQk+is1--7DDzaoOqKMy32VnMMxoFzQ==
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: image

File: config/database.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+
+default: &default
+  adapter: sqlite3
```
Comment (saobv-2223) [inline config/database.yml]: image

File: test/application_system_test_case.rb [hunk 8]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
```
Comment (saobv-2223) [inline test/application_system_test_case.rb]: image

**Comment by xtrxllx** [comment]
ready

File: Gemfile [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+gem "rails", "~> 5.2.3"
+
+gem "sqlite3"
```
Comment (ngocvt-0484) [inline Gemfile]: please use gem mysql2
remove gem sqlite3

**Comment by xtrxllx** [comment]
ready

File: Gemfile (hunk 9 above)
Comment (ngocvt-0484) [reply Gemfile]: you not fix my comment

**Comment by ngocvt-0484** [comment]
commented
 please use gem mysql2

**Comment by xtrxllx** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**